cubi_tk/sodar/lz_list.py

```diff
@@ -62,6 +62,16 @@ def setup_argparse(cls, parser: argparse.ArgumentParser) -> None:
             help="Format string for printing, e.g. %%(uuid)s",
         )
 
+        status = ["ACTIVE", "FAILED", "VALIDATING"]
```

Comment:
These values should probably be defined elsewhere?
I'd say as class attributes, either of this class or in a class that directly interacts with the Sodar landingzone API. Ideally the possible values would even be derived from Sodar or its API somehow, but that might be tricky for use in argparse setup

Reply:
Hm, they're at least not defined in the models in sodar-cli https://github.com/bihealth/sodar-cli/blob/a62505ff9b1365f150bce54c9b2b5e638f245f86/src/sodar_cli/api/models.py#L117

But I will at least move them somewhere else.